01_fib/kasunfdo/main_test.go

```diff
@@ -1,87 +0,0 @@
-package main
```

Comment:
If these changes are needed, they should be in separate PR.

I guess you don't want to remove these files. You should be able to rebase to upstream master and get rid of this modification from git history.

Reply:
lab04 code LGTM

Reply:
those files came from someone committed to real master.